Regeneration check found diff

The post-submit [regeneration check](https://github.com/googleapis/google-cloud-python/actions/runs/26943916519) found a diff.

  Diff summary:
  ```
   packages/google-cloud-storage/tests/unit/conftest.py | 1 +
 1 file changed, 1 insertion(+)
  ```

https://github.com/googleapis/google-cloud-python/actions/runs/27029731047/job/79778765389

```
diff --git a/packages/google-cloud-spanner/noxfile.py b/packages/google-cloud-spanner/noxfile.py
index fa74716b..54ead840 100644
--- a/packages/google-cloud-spanner/noxfile.py
+++ b/packages/google-cloud-spanner/noxfile.py
@@ -43,7 +43,6 @@ UNIT_TEST_STANDARD_DEPENDENCIES = [
     "pytest",
     "pytest-cov",
     "pytest-asyncio",
-    "pytest-xdist",
 ]
 MOCK_SERVER_ADDITIONAL_DEPENDENCIES = [
     "google-cloud-testutils",
@@ -241,8 +240,6 @@ def unit(session, protobuf_implementation):
     # Run py.test against the unit tests.
     args = [
         "py.test",
-        "-n",
-        "auto",
         "-s",
         f"--junitxml=unit_{session.python}_sponge_log.xml",
         "--cov=google",
@@ -757,6 +754,7 @@ def prerelease_deps(session, protobuf_implementation, database_dialect):
 def mypy(session):
     """Run the type checker."""
     session.skip("Mypy is not yet supported")
+
     # TODO(https://github.com/googleapis/gapic-generator-python/issues/2579):
     # use the latest version of mypy
     session.install(
@@ -834,15 +832,12 @@ def core_deps_from_source(session, protobuf_implementation):
     dep_paths = [str(deps_dir / dep) for dep in core_dependencies_from_source]
 
     session.install(*dep_paths, "--no-deps", "--ignore-installed")
-    session.install("pytest-xdist")
     print(
         f"Installed {', '.join(core_dependencies_from_source)} locally from {deps_dir}"
     )
 
     session.run(
         "py.test",
-        "-n",
-        "auto",
         "tests/unit",
         env={
             "PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION": protobuf_implementation,
diff --git a/packages/google-cloud-spanner/setup.py b/packages/google-cloud-spanner/setup.py
index e7dce1a0..34eace7a 100644
--- a/packages/google-cloud-spanner/setup.py
+++ b/packages/google-cloud-spanner/setup.py
@@ -60,17 +60,7 @@ dependencies = [
     "google-cloud-monitoring >= 2.16.0",
     "mmh3 >= 4.1.0",
 ]
-extras = {
-    "libcst": "libcst >= 0.2.5",
-    "test": [
-        "pytest",
-        "mock",
-        "asyncmock",
-        "pytest-cov",
-        "pytest-asyncio",
-        "pytest-xdist",
-    ],
-}
+extras = {"libcst": "libcst >= 0.2.5"}
 
 url = "https://github.com/googleapis/google-cloud-python/tree/main/packages/google-cloud-spanner"
 
diff --git a/packages/google-cloud-storage/tests/system/test_zonal.py b/packages/google-cloud-storage/tests/system/test_zonal.py
index 2d79ec8a..bcdaeaff 100644
--- a/packages/google-cloud-storage/tests/system/test_zonal.py
+++ b/packages/google-cloud-storage/tests/system/test_zonal.py
@@ -27,7 +27,6 @@ from google.cloud.storage.blob import (
     ObjectCustomContextPayload,
 )
 
-
 pytestmark = pytest.mark.skipif(
     os.getenv("RUN_ZONAL_SYSTEM_TESTS") != "True",
     reason="Zonal system tests need to be explicitly enabled. This helps scheduling tests in Kokoro and Cloud Build.",
Error: Process completed with exit code 1.
```

--- a/.librarian/generator-input/client-post-processing/spanner-integration.yaml
+++ b/.librarian/generator-input/client-post-processing/spanner-integration.yaml
@@ -146,7 +146,17 @@ replacements:
           "google-cloud-monitoring >= 2.16.0",
           "mmh3 >= 4.1.0",
       ]
-      extras = {"libcst": "libcst >= 0.2.5"}
+      extras = {
+          "libcst": "libcst >= 0.2.5",
+          "test": [
+              "pytest",
+              "mock",
+              "asyncmock",
+              "pytest-cov",
+              "pytest-asyncio",
+              "pytest-xdist",
+          ],
+      }
     count: 1
   - paths: [packages/google-cloud-spanner/docs/index.rst]
     before: '(?s)API Reference\n-------------.*'
@@ -647,6 +657,7 @@ replacements:
           "pytest",
           "pytest-cov",
           "pytest-asyncio",
+          "pytest-xdist",
       ]
       MOCK_SERVER_ADDITIONAL_DEPENDENCIES = [
           "google-cloud-testutils",
@@ -844,6 +855,8 @@ replacements:
           # Run py.test against the unit tests.
           args = [
               "py.test",
+              "-n",
+              "auto",
               "-s",
               f"--junitxml=unit_{session.python}_sponge_log.xml",
               "--cov=google",
@@ -1358,7 +1371,6 @@ replacements:
       def mypy(session):
           """Run the type checker."""
           session.skip("Mypy is not yet supported")
-
           # TODO(https://github.com/googleapis/gapic-generator-python/issues/2579):
           # use the latest version of mypy
           session.install(
@@ -1400,6 +1412,25 @@ replacements:
       mmh3==4.1.0
       libcst==0.2.5
       googleapis-common-protos==1.60.0
+    count: 1
+  - paths: [
+      packages/google-cloud-spanner/noxfile.py
+    ]
+    before: |
+        session.install\(\*dep_paths, "--no-deps", "--ignore-installed"\)
+        [\s\S]*?session.run\(\s+"py.test",\s+"tests/unit",
+    after: |
+        session.install(*dep_paths, "--no-deps", "--ignore-installed")
+            session.install("pytest-xdist")
+            print(
+                f"Installed {', '.join(core_dependencies_from_source)} locally from {deps_dir}"
+            )
+
+            session.run(
+                "py.test",
+                "-n",
+                "auto",
+                "tests/unit",
     count: 1
   - paths: [packages/google-cloud-spanner/testing/constraints-3.10.txt]
     before: 'grpcio==1.59.0\n(?!grpcio-status)'

